lib/monetize/version.rb

```diff
@@ -1,5 +1,5 @@
 # encoding: utf-8
 
 module Monetize
-  VERSION = '1.7.0'
+  VERSION = '1.8.0'
```

Comment:
I normally manage version releases myself. Can you please remove this change from this PR?